src/All-Movies/All-Movies.scss

```diff
@@ -0,0 +1,21 @@
+@import '../styles.scss';
+
+.movies-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    grid-gap: 10px;
+    background-color: $dark-grey;
+    padding: 20px 0 20px 0;
+    font-size: 150%;
+    height: auto;
+    width: 100vw;
+}
+
+.movies-container>img {
+    height: auto;
+    width: 80%;
+    &:hover {
+        cursor: pointer;
+        box-shadow: 0 0 20px #000;
+    }
```

Comment:
Its not displaying text on the images yet but I figure I could do that within the Movie-Card's styling

Reply:
I think it adds a lot even just with the drop shadow! If you add:
```css
margin: 10px;
width: calc(80% + 10px)
```
to the hover property it will make the image grow too! You could mess around with these values. My idea is to subtract the pixles from the margin and add them to the size with the `calc` method. The `calc` css method is magic if you have never used it! I will include an example of what it would look like with both my suggestion from the previous comment and the properties I am suggesting in this comment. 
<br>
![Rancid Tomatillos (1)](https://user-images.githubusercontent.com/95593550/187080071-7051aeda-88a2-4e68-ba2f-dd8b05a3ed98.gif)
<br>
I didn't add this but you could go a step further and look at the [`transition` property
](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions) to add a smooth transition with growing/size change so it feels less snappy.

Reply:
got it wow! I couldn't figure out how to make them bigger without everything on the page rearranging, this looks so good I can absolutely add all of that!

Reply:
I've pushed up the requested changes!